daemon/graphdriver/devmapper/mount.go

```diff
@@ -6,30 +6,8 @@ import (
 	"bytes"
 	"fmt"
 	"os"
-	"path/filepath"
-
-	"golang.org/x/sys/unix"
 )
 
-// FIXME: this is copy-pasted from the aufs driver.
-// It should be moved into the core.
```

Comment:
Oh, this was added a loooong time ago (https://github.com/moby/moby/commit/4a8a7d4edd53251c8de7ea087b092869e15621d5), and was to satisfy an interface that later changed (https://github.com/moby/moby/commit/b8b509e1c8c9b9def04850eaad8369b9ec6d9e0a / https://github.com/shykes/docker/pull/7). Looks like in aufs this is now moved to `graphdriver.Mounted()` (in https://github.com/moby/moby/commit/65d79e3e5e537039b244afd7eda29e721a93d84f): https://github.com/moby/moby/blob/3a633a712c8bbb863fe7e57ec132dd87a9c4eff7/daemon/graphdriver/aufs/aufs.go#L581-L583